src/screens/formSections/VehicleSection.js

```diff
@@ -1,3 +1,22 @@
+/*
+The VehicleSection is a class component that renders the tiles for a vehicle section on the home screen. This
+includes a tile for the vehicle, a tile for the driver and tiles for any passengers. If the user clicks on the
+tiles, they are navigated to the appropriate form. The component has two views: the regular view, which displays
+the icon buttons, and the edit view, which allows users to delete the vehicle or delete drivers or passengers
+inside the vehicle.
+
+const {navigation, vehicle, index, name, edit, passenger, roadID, questions } = this.props
```

Comment:
What is the point of this line?

Reply:
oops sorry that was for my reference when I was writing out the props